xocto/tracing.py

```diff
@@ -1,9 +1,9 @@
 from __future__ import annotations
 
-import ddtrace
+import ddtrace.trace
 
 
-tracer = ddtrace.tracer
+tracer = ddtrace.trace.tracer
```

Comment:
Thanks for looking at this. 

Is this going to work with Kraken? Kraken is using ddtrace version 2.14.4 https://github.com/octoenergy/kraken-core/blob/b7ca6037788bf6101ef655ba508f4d706fe2df2f/requirements/prod.txt#L142 and in 2.14.4 this code will raise `AttributeError: module 'ddtrace.trace' has no attribute 'tracer'`.

Reply:
In the meantime https://github.com/octoenergy/xocto/pull/199